src/Spectre.libClustering.Tests/PartitionTest.cpp

```diff
@@ -0,0 +1,110 @@
+#define GTEST_LANG_CXX11 1
+
+#include <gtest\gtest.h>
+#include <gtest\gtest-spi.h>
+#include <gsl.h>
+#include <vector>
+#include <algorithm>
+#include "Partition.h"
+
+namespace Spectre::libClustering::Tests
+{
+	class PartitionTest : public ::testing::Test
+	{
+	protected:
+
+		std::vector<unsigned int> testData;
+
+		virtual void SetUp()
+		{
+			testData = { 1, 2, 3, 4, 5, 6, 7, 8 };
+		}
+
+	};
+
+	TEST_F(PartitionTest, NotFailsOnNull)
```

Comment:
I changed to the variant proposed by Grzegorz. Comparing names like:
`comparison_equal_partitions_mixed_labels_no_tolerance` with
`comparison_equal_partitions_mixed_labels_with_tolerance`
seems to be easier than:
`ComparisonEqualPartitionsMixedLabelsNoTolerance` with
`ComparisonEqualPartitionsMixedLabelsWithTolerance`